src/users/pages/Members.vue

```diff
@@ -14,20 +14,63 @@
           :title="$t(sorting === 'joinDate' ? 'GROUP.SORT_NAME' : 'GROUP.SORT_JOINDATE')"
           @click="toggleSorting"
         />
-        <RouterLink
-          v-if="isEditor"
-          :to="{name: 'groupInvitations', params: { groupId }}"
-        >
-          <QBtn
-            small
-            round
-            color="secondary"
-            icon="fas fa-user-plus"
-            :title="$t('GROUP.INVITE_TITLE')"
-          />
-        </RouterLink>
+        <QBtn
+          small
+          round
+          color="secondary"
+          icon="fas fa-user-plus"
+          :title="$t('GROUP.INVITE_TITLE')"
+          @click="inviteLink = true"
+        />
+
+        <QDialog v-model="inviteLink">
```

Comment:
maybe switch to using `CustomDialog` component as this has predefined layout and header styling included? See eg here: https://github.com/yunity/karrot-frontend/pull/2368
I could help if you want, but fine with the current solution, too.

Reply:
Yep! Now using `CustomDialog`. Everything looks fine here, but you might want to double-check